README.md

```diff
@@ -13,33 +13,35 @@ Authors
 Installation
 ------------
 
- 1. Add the PhoneNumberBundle to your dependencies:
-
-        // composer.json
-
-        {
-           // ...
-           "require": {
-               // ...
-               "misd/phone-number-bundle": "~1.0"
-           }
-        }
+ 1. Add the PhoneNumberBundle to your dependencies in the `composer.json`:
+
+```json
```

Comment:
Well I find it less readable but that's just my opinion. So I can let the code blocks if you wish :)